pkg/openstack/actuator.go

```diff
@@ -35,7 +35,8 @@ import (
 )
 
 const (
-	rootOpenStackCredsSecretKey = "clouds.yaml"
+	rootOpenStackCredsSecretKey          = "clouds.yaml"
+	cloudProviderOpenStackCredsSecretKey = "clouds.conf"
```

Comment:
Please add some comments here for the differences between the two keys, as this is definitely going to be confusing.